README.adoc

```diff
@@ -494,6 +492,9 @@ During engine setup, a certificate has to be issued and you will be asked for a
 hostname. If you want to upload and download images from administration portal,
 it has to be the name by which your machine is accessible from the outside.
 
+NOTE: engine-setup might require mod_auth_openidc for keycloak, if so you can enable 
```

Comment:
@arso is this needed for a dev env?

Reply:
If this is needed only for RMP install (as Artur said), then maybe the right place for that is this link?
https://www.ovirt.org/documentation/installing_ovirt_as_a_standalone_manager_with_local_databases/#Installing_the_Red_Hat_Virtualization_Manager_SM_localDB_deploy

Reply:
it's already there https://github.com/oVirt/ovirt-site/pull/2835

Reply:
In that case this change should be reverted.

Reply:
mod_auth_openidc is required only for ovirt-engine-keycloak, which is not useable in development environment. If you need Keycloak integration for your development, then you need to develop on RPM installation